src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/LanguageServices/UnconfiguredProjectHostObject.cs

```diff
@@ -11,16 +11,12 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.LanguageServices
     /// </summary>
     internal sealed class UnconfiguredProjectHostObject : AbstractHostObject, IUnconfiguredProjectHostObject
     {
-        private readonly Dictionary<uint, IVsHierarchyEvents> _hierEventSinks;
-        private readonly HashSet<uint> _pendingItemIds;
+        private readonly Dictionary<uint, IVsHierarchyEvents> _hierEventSinks = new Dictionary<uint, IVsHierarchyEvents>();
+        private readonly HashSet<uint> _pendingItemIds = new HashSet<uint>();
 
-        public UnconfiguredProjectHostObject(UnconfiguredProject unconfiguredProject)
-            : base (innerHierarchy: (IVsHierarchy)unconfiguredProject.Services.HostObject, innerVsProject: (IVsProject)unconfiguredProject.Services.HostObject)
+        public UnconfiguredProjectHostObject(IUnconfiguredProjectVsServices projectVsServices)
+            : base(innerHierarchy: projectVsServices.VsHierarchy, innerVsProject: projectVsServices.VsProject)
```

Comment:
Yep, looks like a good sign that I should add context in the commit for the future.